client/verta/tests/http_requests/test_client.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
```

Comment:
Most of this file is copied from `test_entities.py`; scroll to the bottom of this file to see the new test.